tests/parser.js

```diff
@@ -341,5 +341,135 @@ describe('parser', () => {
             expect(result[3].tournamentPlace).toBe(3);
             expect(result[4].tournamentPlace).toBe(4);
         });
+
+        describe('(when parsing player names)', function () {
+            it('find player name by provided column number', function () {{
+                const result = testMap(
+                    `<table>
+                        <tr><th>Place</th><th>Name</th></tr>
+                        <tr><td>1</td><td>Player 1</td></tr>
+                        <tr><td>2</td><td>Player 2</td></tr>
+                        <tr><td>3</td><td>Player 3</td></tr>
+                        <tr><td>4</td><td>Player 4</td></tr>
+                    </table>`, {
+                        nameColumns : 1
+                    });
+                expect(result[1].name).toBe("Player 1");
```

Comment:
Coding conventions: use single quotes instead of double quotes (and enable `eslint`)